RFS/src/main/java/org/opensearch/migrations/bulkload/common/http/ConnectionContext.java

```diff
@@ -87,8 +89,10 @@ private ConnectionContext(IParams params) {
             requestTransformer = new BasicAuthTransformer(params.getUsername(), params.getPassword());
         }
         else if (sigv4Enabled) {
+            var credProvider = DefaultCredentialsProvider.builder().build();
+            log.info("Sigv4 enabled with {}, aws service: {}, region: {}", credProvider.resolveCredentials().toString(), params.getAwsServiceSigningName(), params.getAwsRegion());
```

Comment:
Added this to assist in troubleshooting authentication issues (it helped a little)